go.mod

```diff
@@ -45,3 +45,5 @@ require (
 	google.golang.org/grpc v1.46.2
 	gopkg.in/yaml.v3 v3.0.1
 )
+
+replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.2
```

Comment:
I consider this a temporary fix, not a fundamental fix.
Would you please leave a comment regarding when this exceptional statement can be removed?

Reply:
Yes. I added a comment.

Reply:
Thank you for indicating sufficient conditions to remove this temporary fix. Additionally, could you please change it to publicly available information since the last URL is a private URL as noted in the PR comment?
(Comments on the source aren't linked, so just the CVE or JVN number is fine, not the URL.)

Reply:
I fixed it.

Reply:
Thanks, I'll approve.